### PR TITLE
model-index-js

### PR DESCRIPTION
**Pull request by elizabethkim828**
revised associations

### CODE REVIEW

File: server/db/index.js [hunk 1]
```diff
@@ -6,13 +6,12 @@ var Review = require('./models/review');
 var User = require('./models/user')(db);
 var Order = ('./models/order')(db);
 
-Kit.hasMany(Item)
+Kit.belongsToMany(Item)
```
Comment (jdhang) [inline server/db/index.js:9]: Nice!

File: server/db/index.js [hunk 2]
```diff
@@ -6,13 +6,12 @@ var Review = require('./models/review');
...
 Kit.hasMany(Review)
-Kit.hasMany(Order)
+Kit.belongsToMany(Order)
```
Comment (jdhang) [inline server/db/index.js:11]: Smashing!

**Comment by jdhang** [comment]
This looks good to me, @ARRiera96, what do you think?

**Comment by ARRiera96** [comment]
@jdhang I'm not impressed at all...

**Pull request merged**